src/libraries/System.Private.CoreLib/src/System/Buffers/IMemoryOwner.cs

```diff
@@ -3,14 +3,20 @@
 
 namespace System.Buffers
 {
-    /// <summary>
-    /// Owner of Memory<typeparamref name="T"/> that is responsible for disposing the underlying memory appropriately.
-    /// </summary>
+#pragma warning disable CS1574 // XML comment has cref attribute that could not be resolved (System.Buffers.MemoryPool{T}.Rent)
```

Comment:
This was a little interesting. `System.Buffers.MemoryPool<T>` isn't visible within the `System.Buffers` build because of it coming from `System.Private.CoreLib`. Retaining the `<see cref>` reference to it seemed valuable though. If we see enough of these situations, it would be _possible_ for the DocsPortingTool to detect this situation and apply the `#pragma` automatically.